src/Components/CarsMain/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PopularCar from '../CarGrid'
+import { Button, Grid, Typography } from '@mui/material'
+export const CarsMain = () => {
+    const PopularCars = [
```

Comment:
move data to a seperate file

Reply:
this is only a mock data to test the functionality of the car card.